components/feature/session/src/main/java/mozilla/components/feature/session/SessionUseCases.kt

```diff
@@ -200,17 +200,21 @@ class SessionUseCases(
     ) {
         /**
          * Navigates back in the history of the currently selected tab
```

Comment:
It's possible that my above comment should really apply to the use case. If so, can we also add a period to the end of this comment so it's consistent throughout the file.

Reply:
I tried adding a note here but they all end up being explanations of the Gecko internals.

I'm open to recommendations though.

Reply:
See above.